clang/lib/InstallAPI/Visitor.cpp

```diff
@@ -255,7 +255,7 @@ bool InstallAPIVisitor::VisitFunctionDecl(const FunctionDecl *D) {
       return true;
 
     // Skip methods in CXX RecordDecls.
-    for (auto P : D->getASTContext().getParents(*M)) {
+    for (const auto &P : D->getASTContext().getParents(*M)) {
```

Comment:
```suggestion
    for (const DynTypedNode &P : D->getASTContext().getParents(*M)) {
```
An improvement would be to be explicit about the type.

Reply:
Thanks you @cyndyishida for the reviews. I have updated patch to be explicit about the type and the title.